apps/televisit-demo/backend/README.md

```diff
@@ -0,0 +1,56 @@
+# Amazon Chime SDK Televisit Demo: Backend Infrastructure
+
+The diagram of Architecture is here:
+
+![arch](images/chime-sdk-telemedicine.jpg)
```

Comment:
The architecture diagram needs to reflect correct service names. eg. `AWS Lambda` is the name of the service, not `Amazon Lambda`